src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -1604,7 +1600,7 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 	private doRedrawTabActive(isGroupActive: boolean, allowBorderTop: boolean, editor: EditorInput, tabContainer: HTMLElement, tabActionBar: ActionBar): void {
 
 		const isActive = this.tabsModel.isActive(editor);
-		const isSelected = this.groupView.isSelected(editor); // TODO, move to model
+		const isSelected = this.groupView.isSelected(editor); // TODO, move to model (whats this?)
```

Comment:
Is this an oversight?

Reply:
Jup, wanted to use readonly model here `tabsModel`